File: LightningDeath/src/main/java/me/provsalt/lightningdeath/Loader.java
package me.provsalt.lightningdeath;

import cn.nukkit.plugin.PluginBase;
import me.provsalt.lightningdeath.events.Death;

public class Loader extends PluginBase {
    @Override
    public void onEnable(){
        saveDefaultConfig();
        getServer().getPluginManager().registerEvents(new Death(this), this);
    }
}

File: src/provsalt/lightningdeath/Loader.php
<?php

namespace provsalt\lightningdeath;

use pocketmine\plugin\PluginBase;
use JackMD\UpdateNotifier\UpdateNotifier;

class Loader extends PluginBase {

    public function onEnable() :void{
        $this->getServer()->getPluginManager()->registerEvents(new LightningListener($this), $this);
        if ($this->getConfig()->get("version") !== 1){
            $this->getLogger()->critical("Please regenerate your config file!");
            $this->getServer()->getPluginManager()->disablePlugin($this);
        }
        UpdateNotifier::checkUpdate($this->getDescription()->getName(), $this->getDescription()->getVersion());
    }
}
